Makefile.am

```diff
@@ -154,6 +154,8 @@ cpplint_FILES += $(noinst_HEADERS)
 clean-local:
 	-rm -f $(BUILT_SOURCES) $(CLEANFILES)
 
+check: cpplint
+
```

Comment:
stupid cpplint writes the message on stderror.
Want me to change the python file?